### ISSUE TITLE
"More options" button of "Untitled" button appearing at the left side of the screen is not accessible through keyboard

### ISSUE THREAD

**Issue by melode11** [SeleniumHQ/selenium-ide#1184]
## User Experience
Keyboard user is not able to access the controls and might miss the functionality if the controls are not accessible through keyboard.

## Repro Steps
Open Selenium IDE.
Navigate to "More options" button of "Untitled" button appearing at the left side of the screen
Verify the issue.

## Actual Result
"More options" button of "Untitled" button appearing at the left side of the screen is not accessible through keyboard due to which keyboard user is not able to access the controls present under the button
## Expected Result
"More options" button of "Untitled" button appearing at the left side of the screen is not accessible through keyboard

## NOTE
- Tabstops feature is not working on it
- Same issue also occurs with "More options" button of table rows 
- Same issue also occurs with "Create new project" button[MAS2.1.1_Create a new project button is not accessible through keyboard]

## MAS Reference
MAS 2.1.1 - Keyboard (36)

## Environment

OS: Windows 10
Selenium IDE Version:  3.17.0
Browser: Chrome
Browser Version: 87.0.4280.141

**Comment by melode11** [comment]
PR merged, close issue.
